Form/Form.js

```diff
@@ -1,23 +1,32 @@
 import React from 'react'
+import Empty from './Empty'
 
 const Form = (props) => {
-  return (
-    <div class="card-container card" >
-      <div class="card-body">
-        <form className="user-form" onSubmit={props.submit}>
-          <h1 className="title">Create a Flash Card</h1>
-          <div className="form-group">
-            <label>Question</label>
-            <input type="text" value={props.value.question} onChange={props.change} name="question" className="form-control" id="user-question-input" placeholder="Enter Question"/>
-          </div>
-          <div className="form-group">
-            <label>Answer</label>
-            <input type="text" value={props.value.answer} onChange={props.change} name="answer" className="form-control" id="user-answer-input" placeholder="Enter Answer"/>
-          </div>
-          <button type="submit" className="save-btn btn btn-primary">Save</button>
-        </form>
-      </div>
+  let toBeRendered = <Empty click={props.click}/>;
+  const counter = `Number of Flashcards: ${props.value.data.length}`
```

Comment:
This counter doesn't really belong here. Maybe you can include the count of cards in your navigation link?